framework/editor/InputTarget.kt

```diff
@@ -169,7 +169,8 @@ internal class InputTarget constructor(
         val relY = y - textAreaBounds.top + verScroller.offset.value
         val row = floor(relY / lineHeight.value).toInt().coerceIn(0, lineCount - 1)
         val offsetInLine = Offset(relX * density, (relY - (row * lineHeight.value)) * density)
-        val col = rendering.get(row)?.getOffsetForPosition(offsetInLine) ?: 0
+        val charOffset = rendering.get(row)?.getOffsetForPosition(offsetInLine) ?: 0
```

Comment:
As a part of our workarounds, we need to translate from Glyph offsets to Char offsets. Our internal `Cursor` and `Selection` classes now use Glyph offsets, but we have to translate to Char and from Char when giving data to or getting data from Compose library functions respectively.